backend/src/utils/error.rs

```diff
@@ -33,14 +41,18 @@ impl error::ResponseError for Error {
             Error::Unauthorized => StatusCode::FORBIDDEN,
             Error::PasswordAuthNotAvailable => StatusCode::FORBIDDEN,
             Error::UserAlreadyExists => StatusCode::CONFLICT,
+            Error::TeamAlreadyExists => StatusCode::CONFLICT,
```

Comment:
you can merge identical returns
```suggestion
            Error::TeamAlreadyExists | Error::UserAlreadyExists => StatusCode::CONFLICT,
```